js/reducers/currentbuild.js

```diff
@@ -0,0 +1,26 @@
+var _ = require('underscore');
+
+var actions = require('../_actions.js');
+
+module.exports = function currentbuild(state, action) {
+	if(typeof state === 'undefined') {
+		return {
+			build: [],
+			is_loading: false
+		};
+	}
+
+	switch (action.type) {
+		case actions.START_CURRENT_BUILD_STATUS_GET:
+			return _.assign({}, state, {
+				is_loading: true
+			});
+		case actions.SUCCEED_CURRENT_BUILD_STATUS_GET:
+			return _.assign({}, state, {
+				build: action.data.build,
+				is_loading: false
+			});
```

Comment:
There is no error handling for either of these components, so probably good to look at that next. :)